Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -585,6 +645,51 @@ instance : CommMonoid R[S⁻¹] where
 
 end CommMonoid
 
+section Zero
+
+variable {R : Type*} [Monoid R] {S : Submonoid R} [OreSet S] {X : Type*} [Zero X]
+variable [MulAction R X]
+
+private irreducible_def zero (lemma := zero_def') : X[S⁻¹] := 0 /ₒ 1
```

Comment:
I think we prefer the `irreducible` tag now and then we can use `unseal`/`with_unfolding_all` to access the internals.